custom_components/waste_collection_schedule/waste_collection_schedule/source/info_collectes_ca.py

```diff
@@ -0,0 +1,126 @@
+from datetime import datetime
+from waste_collection_schedule.exceptions import SourceArgumentException, SourceArgumentNotFound
+from waste_collection_schedule import Collection
+import requests
+import unicodedata
+
+TITLE = "MRC de Roussillon (QC)" # Title will show up in README.md and info.md
+DESCRIPTION = "Source script for info-collectes.ca/"  # Describe your source
+URL = "https://info-collectes.ca/"  # Insert url to service homepage. URL will show up in README.md and info.md
+TEST_CASES = {  # Insert arguments for test cases to be used by test_sources.py script
+    "TestName1": {"municipality": 'La Prairie'},
+    "TestName2": {"municipality": "candiac", "sector": "est"},
+    "TestName3": {"municipality": "chateauguay", "sector": "est"},
+    "TestName4": {"municipality": "chateauguay", "sector": "sud"},
+    "TestName5": {"municipality": 'lery'},
+}
+
+API_URL = "https://info-collectes.ca/wp/wp-admin/admin-ajax.php"
+ICON_MAP = {   # Optional: Dict of waste types and suitable mdi icons
+    "organic": "mdi:compost",
+    "recycling": "mdi:recycle",
+    "greenWaste": "mdi:leaf",
+    "cardboard": "mdi:package-variant",
+    "bulky": "mdi:sofa",
+    "garbage": "mdi:truck-remove"
+}
+
+MUNICIPALITY_NAMES = ["candiac",
+                      "saint-constant",
+                      "chateauguay",
+                      "saint-isidore",
+                      "delson",
+                      "saint-mathieu",
+                      "la-prairie",
+                      "saint-philippe",
+                      "lery",
+                      "sainte-catherine",
+                      "mercier",]
```

Comment:
If you only allow thse municipalites anyway you can typehint the parameter to be a `typing.Literal` this way HA will show a drop down menu of these instead of a text input box


```suggestion
MUNICIPALITY_LITERAL = Literal[
    "candiac",
    "saint-constant",
    "chateauguay",
    "saint-isidore",
    "delson",
    "saint-mathieu",
    "la-prairie",
    "saint-philippe",
    "lery",
    "sainte-catherine",
    "mercier",
]

MUNICIPALITY_NAMES = list(get_args(MUNICIPALITY_LITERAL))
```